barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mock_circuits.hpp

```diff
@@ -19,13 +19,22 @@ class MockCircuits {
      * @param num_gates
      */
     template <typename Builder>
-    static void add_arithmetic_gates_with_public_inputs(Builder& builder, const size_t num_gates = 4)
+    static void add_arithmetic_gates_with_public_inputs(Builder& builder,
+                                                        const size_t num_gates = 4,
+                                                        bool random = true)
     {
         // For good measure, include a gate with some public inputs
         for (size_t i = 0; i < num_gates; ++i) {
-            FF a = FF::random_element(&engine);
-            FF b = FF::random_element(&engine);
-            FF c = FF::random_element(&engine);
+            FF a, b, c;
```

Comment:
Unless you're confident you're going to use this again, will you revert all of this?